database/upgrades.go

```diff
@@ -55,6 +55,9 @@ type v1Ticket struct {
 	FeeTxHash         string            `json:"fhsh"`
 	FeeTxStatus       FeeStatus         `json:"fsts"`
 	Outcome           TicketOutcome     `json:"otcme"`
+	// AltSigAddress was added later. Here to allow converting to the new
+	// type.
+	AltSigAddress string
```

Comment:
Adding this field to `v1Ticket` means it is no longer representative of a real version 1 ticket. We can avoid adding this field if we change the conversion in `upgrade_v3.go` to be explicit, ie:

```
newTicket := Ticket{
    Hash: v1Ticket.Hash,
    ...
}

```

This will also benefit my upcoming PR which adds consensus vote choices to `Ticket`, and any other future work which adds new fields.

Reply:
Did this I think. 6e98b01